src/components/Requests/Request/Request.scss

```diff
@@ -0,0 +1,117 @@
+@import "src/constants/style";
+
+$transition-time: 0.2s;
+$transition-time--text: 0.1s;
+$fly-in-time: 0.5s;
+
+// CONTAINER
+.request__container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  column-gap: 12px;
+
+  width: auto;
+  height: 56px;
+
+  padding: 0 5px;
+  border-radius: 28px;
+
+  animation: fly-in-right $fly-in-time ease-out;
+
+  background-color: $color-white;
+  box-shadow: 0 3px 5px rgba($menu-icon-background-color--dark, 30%);
+
+  transition: margin-right $transition-time ease-out, padding-right $transition-time ease-out;
+
+  &:hover {
+    padding-right: 10px;
+    margin-right: 10px;
```

Comment:
Why padding and margin?
Wouldn't margin be enough? And could you use $padding-default or $padding-small?

Reply:
Not sure whether I get what you mean. The :hover pseudo class had only the two right spacings and I just wondered why you need to push the outer _and_ inner box instead of just the outer (`margin-right: 20px;`) or the inner (`padding-right: 20px;`). That was why I asked :)
The effect per se is nice :)